README.txt

```diff
@@ -22,3 +22,21 @@ Gallerya transforms the WordPress native post gallery into a full fledged slides
 
 * PHP 7.0 or later.
 
+== Contributors & Developers ==
```

Comment:
These are stated in the readme.txt header, not in a custom text section.

Please have a look at https://github.com/Mollom/wordpress-mollom/blob/master/readme.txt (and compare with https://wordpress.org/plugins/mollom/) to see how the readme.txt file works.

We should only state people that (1) have a wp.org account and (2) contributed significantly to appear in the authors/maintainers section (in the sidebar).